tests/integration/src/test/java/org/mule/test/config/ExceptionHelperTestCase.java

```diff
@@ -80,9 +85,70 @@ public void testGetNonMuleExceptionCause()
         assertThat(ExceptionHelper.getNonMuleException(new IllegalArgumentException()),IsInstanceOf.instanceOf(IllegalArgumentException.class));
     }
 
+    @Test
+    public void testFilteredStack1()
```

Comment:
Don't use testFilteredStack1, testFilteredStack2, etc. Use test names that describe the difference between them